<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionKey</class>
 <widget class="QWidget" name="FunctionKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;
color:white;
font-family:&quot;맑은 고딕&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	border:none;
}</string>
   </property>
   <property name="text">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	border:none;
}</string>
   </property>
   <property name="text">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	border:none;
}</string>
   </property>
   <property name="text">
    <string>F3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	border:none;
}</string>
   </property>
   <property name="text">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(76, 143, 255, 255), stop:1 rgba(255, 143, 133, 255));
	border:none;
}</string>
   </property>
   <property name="text">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 229, 129, 255), stop:1 rgba(255, 109, 96, 223));
	border:none;
	color:black;
}</string>
   </property>
   <property name="text">
    <string>F6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 229, 129, 255), stop:1 rgba(255, 109, 96, 223));
	border:none;
	color:black;
}</string>
   </property>
   <property name="text">
    <string>F7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 229, 129, 255), stop:1 rgba(255, 109, 96, 223));
	border:none;
	color:black;
}</string>
   </property>
   <property name="text">
    <string>F8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 229, 129, 255), stop:1 rgba(255, 109, 96, 223));
	border:none;
	color:black;
}</string>
   </property>
   <property name="text">
    <string>F9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#494949;
}

QPushButton:hover{
	background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 229, 129, 255), stop:1 rgba(255, 109, 96, 223));
	border:none;
	color:black;
}</string>
   </property>
   <property name="text">
    <string>F10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
